client/styles/components/_components.side-bar.scss

```diff
@@ -1,7 +1,7 @@
 .c-sidebar{
     position: relative;
     width: 140px;
-    min-width: 140px;
+    // min-width: 140px;
```

Comment:
If you don't need this comment then take it out. But if you think you might need it later then it is good to stay.